index.html

```diff
@@ -200,7 +200,7 @@ <h2 class="prt2">Multi-Post Stories</h2>
           </li>
         </ul>
         <div class="btl">
-          <button class="btn"><a href="#" class="btt">See Project</a></button>
+          <button class="btn"><a href="" class="btt">See Project</a></button>
         </div>
```

Comment:
- [ ] When I click on this `see project` button, it shows the popup modal but the page refreshes after some seconds. I think this is because of the `a` tag you used here so it is better to use only the `button` tag. :relaxed:

Reply:
The issue has been resolved. Thanks.